feathr_project/feathr/client.py

```diff
@@ -928,19 +929,32 @@ def _collect_secrets(self, additional_secrets=[]):
             prop_and_value[prop] = self.envutils.get_environment_variable_with_default(prop)
         return prop_and_value
 
-    def get_features_from_registry(self, project_name: str) -> Dict[str, FeatureBase]:
+    def get_features_from_registry(self, project_name: str, return_keys: bool = False, verbose: bool = False) -> Dict[str, FeatureBase]:
```

Comment:
The returned type should also be changed accordingly

Reply:
Changed as below mentioned